R/upkeep.R

```diff
@@ -118,10 +118,12 @@ upkeep_checklist <- function(target_repo = NULL) {
 
 #' @export
 #' @rdname tidyverse
-#' @param year Approximate year when you last touched this package. If `NULL`,
-#'   the default, will give you a full set of actions to perform.
-use_tidy_upkeep_issue <- function(year = NULL) {
-  make_upkeep_issue(year = year, tidy = TRUE)
+#' @param year Approximate year when you last touched this package. Default will
```

Comment:
I suspect few people supply this argument by name, so we could make it less confusing by calling it `last_upkeep`?

Reply:
yeah... also, it dawned on me that everyone is calling these functions interactively (I hope) - changing argument names is basically a non-issue afaik